src/checkbox.page/checkbox/checkbox.types.js

```diff
@@ -0,0 +1,11 @@
+// @flow
```

Comment:
I would recommend putting the type next to the component you're defining to keep it as contextualized as possible. Its up to  you tho.